op-guide/docker-across-multiple-hosts.md

```diff
@@ -12,11 +12,15 @@ Assume we have 3 machines with the following details:
 |**host2**|192.168.1.101|
 |**host3**|192.168.1.102|
 
-Every host has already installed newest version docker, and pulled the latest docker images of TiDB/TiKV/PD.
+On each host, we have installed Docker with the latest version and pulled the latest Docker images of TiDB, TiKV and PD.
 
-## Start `busybox` container as storage volume for each host
+`docker pull pingcap/tidb:latest`
```

Comment:
Please adjust the format because now they are on one line. On a second thought, maybe we don't need to list these commands. What do you think? @shenli